README.md

```diff
@@ -1,24 +1,29 @@
-## Example Kit Integration
+## PilgrimSDK Kit Integration
 
-[See here for more information](https://github.com/mParticle/mparticle-android-sdk/wiki/Kit-Development) on how to use this example to write a new kit.
+This repository contains the [PilgrimSDK](https://enterprise.foursquare.com/products/pilgrim) integration for [mParticle Android SDK](https://github.com/mParticle/mparticle-android-sdk).
```

Comment:
given the android audience, might make sense to link to the android quick start section of our docs